lib/fluent/plugin/in_watch_process.rb

```diff
@@ -1,4 +1,5 @@
 require 'time'
+require 'csv'
```

Comment:
`require 'csv' if Fluent.windows?` may be more better because it's not used on other platforms.

Reply:
Thank you for the advice.
I will fix it.

Reply:
I have fixed this in https://github.com/daipom/fluent-plugin-watch-process/pull/2/commits/e0b318de2dff6c286c1e4e43eefcb39eae77ac84.